util.py

```diff
@@ -301,3 +302,43 @@ def instanciate_delayed(o):
         for k in o:
             o[k] = instanciate_delayed(o[k])
     return o
+
+
+def get_executable_path(
+    path: tk.Path, gs_member_name: str, default_exec_path: Optional[tk.Path] = None
```

Comment:
`path` and `gs_member_name` are also `Optional` as it is fully intended that they can be `None`, also maybe this function here could get a docstring.

Reply:
`path` is Optional now. I left the `gs_member_name` as currently I do not support _not_ giving it.

It would be easy to add this support, but for all paths with no defined `gs_member_name` we don't need this function in the first place